stable/zalenium/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: zalenium
+version: 0.0.1
+appVersion: 3.141.59
+description: Chart for deploying Zalenium
+sources:
+  - https://github.com/zalando/zalenium
+maintainers:
+  - name: diemol
+    email: diemol@gmail.com
+  - name: FFX Blue Operations
+    email: blueops@fairfaxmedia.com.au
```

Comment:
I believe there's a github email to use - take a look in the other charts in this repo...

Reply:
I actually got these details from `confidant` chart.. And looking now at other charts I see that they use a dufferent email. Will update accordingly.